docs/gadget.md

```diff
@@ -30,3 +30,6 @@ partition as `grub.cfg` and `uboot.env`. The bootloader configuration
 contains the boot logic. Examples for the boot logic can be found in
 the `pc` and the `pi2` gadget snaps.
 
+A cloud-init configuration may be provided at the toplevel of the
+gadget snap as `cloud.conf`. It will be copied into the image as
```

Comment:
Because that's consistent on our end. Gadget has:
```
cloud.conf
u-boot.conf
grub.conf
```
That said, I would prefer to not mention the implementation of it, stopping after cloud.conf above. We can change where this goes without it being a problem for people.